src/styles/order-service.less

```diff
@@ -50,9 +50,8 @@
       font-weight: 600;
       color: @color-pf-black-600;
     }
-    .version-dropdown {
-      padding-top: 28px;
-      cursor: pointer;
+    .builder-description {
+      white-space: pre-wrap;
```

Comment:
curious what is different about the builder description compared with a service class description?